### ISSUE TITLE
Both installation methods fail on Mac (M2)

### ISSUE THREAD

**Issue by rowild** [ltdrdata/ComfyUI-Impact-Pack#408]
macOs 13.6

Installation via `python install.py` throws these errors:

```sh
ComfyUI-Impact-Pack git:(Main) git submodule update --init --recursive                      
➜  ComfyUI-Impact-Pack git:(Main) python install.py                      
/Users/robertwildling/.pyenv/versions/3.10.13/lib/python3.10/site-packages/transformers/utils/generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
### ComfyUI-Impact-Pack: Check dependencies
### ComfyUI-Impact-Pack: Updating subpack
[ERROR] ComfyUI-Impact-Pack: Dependency installation has failed. Please install manually.
Traceback (most recent call last):
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 281, in <module>
    install()
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 235, in install
    ensure_subpack()  # The installation of the subpack must take place before ensure_pip. cv2 triggers a permission error.
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 115, in ensure_subpack
    import git
ModuleNotFoundError: No module named 'git'
```

After deleting, cloning again and then install dependencies manually, errors look like this:

```sh
ComfyUI git:(master) ✗ python main.py --force-fp16
Total VRAM 32768 MB, total RAM 32768 MB
Forcing FP16.
/Users/robertwildling/.pyenv/versions/3.10.13/lib/python3.10/site-packages/transformers/utils/generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
/Users/robertwildling/.pyenv/versions/3.10.13/lib/python3.10/site-packages/transformers/utils/generic.py:309: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
Set vram state to: SHARED
Device: mps
VAE dtype: torch.float32
Using sub quadratic optimization for cross attention, if you have memory or speed issues try using: --use-split-cross-attention
### Loading: ComfyUI-Impact-Pack (V4.56)
### ComfyUI-Impact-Pack: Updating dependencies [0 -> 20]
### ComfyUI-Impact-Pack: Check dependencies
### ComfyUI-Impact-Pack: Updating subpack
[ERROR] ComfyUI-Impact-Pack: Dependency installation has failed. Please install manually.
Traceback (most recent call last):
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 281, in <module>
    install()
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 235, in install
    ensure_subpack()  # The installation of the subpack must take place before ensure_pip. cv2 triggers a permission error.
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 115, in ensure_subpack
    import git
ModuleNotFoundError: No module named 'git'
### ComfyUI-Impact-Pack: Reinstall dependencies (several dependencies are missing.)
### ComfyUI-Impact-Pack: Check dependencies
### ComfyUI-Impact-Pack: Updating subpack
[ERROR] ComfyUI-Impact-Pack: Dependency installation has failed. Please install manually.
Traceback (most recent call last):
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/__init__.py", line 49, in <module>
    import impact.subpack_nodes  # This import must be done before cv2.
ModuleNotFoundError: No module named 'impact.subpack_nodes'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 281, in <module>
    install()
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 235, in install
    ensure_subpack()  # The installation of the subpack must take place before ensure_pip. cv2 triggers a permission error.
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/install.py", line 115, in ensure_subpack
    import git
ModuleNotFoundError: No module named 'git'
[WARN] ComfyUI-Impact-Pack: `ComfyUI` or `ComfyUI-Manager` is an outdated version.
### ComfyUI-Impact-Pack: (IMPORT FAILED) Subpack

  The module at the `custom_nodes/ComfyUI-Impact-Pack/impact_subpack` path appears to be incomplete.
  Recommended to delete the path and restart ComfyUI.
  If the issue persists, please report it to https://github.com/ltdrdata/ComfyUI-Impact-Pack/issues.

---------------------------------
Traceback (most recent call last):
  File "/Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack/__init__.py", line 456, in <module>
    import impact.subpack_nodes
ModuleNotFoundError: No module named 'impact.subpack_nodes'
---------------------------------


Import times for custom nodes:
   7.1 seconds: /Users/robertwildling/Desktop/_AI/ComfyUI/custom_nodes/ComfyUI-Impact-Pack

Starting server

To see the GUI go to: http://127.0.0.1:8188
[Impact Pack] Wildcards loading done. // <== this line appears only after a while
```

Am I doing sth wrong here? Do I miss sth?

**Comment by ltdrdata** [comment]
Update Impact Pack and try again.
I just patched install script.

**Comment by rowild** [comment]
Thank you! I think it is working. 
What I did before was running the install script manually. But it seems that after cloning all that should be done is starting ComfyUI again and the start-up process will catch the new node and perform the updates. This is what I did now and it works fine as it seems.
Maybe it should be mentioned in the Readme that the installation of requirements happens on ComfyUi restart and no manual action is required?
